context/jest.config.js

```diff
@@ -2,11 +2,16 @@ module.exports = {
   testPathIgnorePatterns: ['jest.config.js', '/node_modules/', '/cypress/'],
   setupFilesAfterEnv: ['<rootDir>/test-utils/setupTests.js'],
   moduleNameMapper: {
+    '\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$':
+      '<rootDir>/test-utils/__mocks__/file.js',
     '^.+\\.css$': '<rootDir>/test-utils/__mocks__/style.js',
     '^components(.*)$': '<rootDir>/app/static/js/components$1',
     '^static(.*)$': '<rootDir>/app/static/$1',
     '^test-utils(.*)$': '<rootDir>/test-utils$1',
     '^helpers(.*)$': '<rootDir>/app/static/js/helpers/$1',
+    '^hooks(.*)$': '<rootDir>/app/static/js/hooks/$1',
+    '^shared-styles(.*)$': '<rootDir>/app/static/js/shared-styles/$1',
+    '^theme$': '<rootDir>/app/static/js/theme.jsx',
```

Comment:
Not a blocker for now, but would it be possible to capture everything under `<rootDir>/app/static/js` without spelling out each subdirectory? I worry about someone reorganizing or adding a new directory down the road, and needing to remember to update this list.

Reply:
Yes, I definitely should. I'll clean things up before merging. I wanted to keep things parallel to our webpack config (which needs similar cleaning).